### ISSUE TITLE
Can we get ProcessID/AppName for each of incoming request?

### ISSUE THREAD

**Issue by TheMrugraj** [apify/proxy-chain#248]
Hello,

Is there any possibility to identify the Client App which has made particular request?

For example:
If user has accessed apple.com from Safari, can we identify App name/App ID/ProcessID of Safari to capture usage of source Client ? The idea is to track the usage by App.

Thanks in Advance!

**Comment by fnesveda** [comment]
Not really, you can't get the process ID of the calling process when you receive a request from it.
